includes/api/ApiQueryUserContributions.php

```diff
@@ -59,25 +59,10 @@ public function execute() {
 		// TODO: if the query is going only against the revision table, should this be done?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)

[project-id]: wikia:app